Description: Add an explicit dependency on xpp3 to avoid a runtime exception
 (xstream < 1.4 did depend on xpp3, but it became optional in the version 1.4)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/maven-autotag-plugin/pom.xml
+++ b/maven-autotag-plugin/pom.xml
@@ -65,6 +65,11 @@
     	<version>1.3.1</version>
     </dependency>
     <dependency>
+    	<groupId>xpp3</groupId>
+    	<artifactId>xpp3</artifactId>
+    	<version>debian</version>
+    </dependency>    
+    <dependency>
     	<groupId>org.apache.tiles</groupId>
     	<artifactId>tiles-autotag-jsp</artifactId>
     	<version>1.2</version>
